frontend/src/App.tsx

```diff
@@ -27,15 +28,15 @@ function App() {
         <StyledToolbar />
         <Routes>
           <Route path="/" element={<HomePage />} />
-          <Route path="/signup" element={<SignUpPage />} />
+          <Route path="/signup" element={<SignupPage/>} />
           <Route path="/login" element={<LoginPage />} />
           <Route
-            path="/expenses/friendslist"
-            element={<ProtectedRoute element={<FriendsListPage />} />}
+            path="/expense/selectfriends"
```

Comment:
For url path, camel case is not common.

https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris/18449772#18449772

https://developers.google.com/search/docs/crawling-indexing/url-structure?hl=en&visit_id=638236451483817885-4118748611&rd=1

Reply:
Okay. Changed it.